### ISSUE TITLE
blender fails to build (crux 3.2)

### ISSUE THREAD

**Issue by rmull** [6c37/crux-ports#24]
here's the buildlog: http://sprunge.us/ZYSe

Might be related to #23 because openimageio is a blender dep, but the errors didn't look related to me. OpenMP is mentioned in the gcc5 docs so I'm wondering if the failure is due to the move to gcc5.

**Comment by onodera-punpun** [comment]
I'm on holiday right now, I'll try fixing the port when I'm back, which will be in January. If anyone else can try fixing/updating the port I will hugely appreciate it.

**Comment by rmull** [comment]
https://developer.blender.org/T46464

Looks like this might be fixed with python 3.5.1, which was released Dec 7. See #29.

**Comment by rmull** [comment]
Build failure is resolved by #29, but it segfaults on startup:
```
$ blender
Color management: using fallback mode for management
Ignoring Xlib error: error code 3 request code 154
GLEW Error (0x0001): GLEW_ERROR_NO_GL_VERSION: Missing GL version
GLEW Error (0x0001): GLEW_ERROR_NO_GL_VERSION: Missing GL version
Writing: /tmp/blender.crash.txt
Segmentation fault
```

And the crash file:
```
$ cat /tmp/blender.crash.txt 
# Blender 2.76 (sub 0), Commit date: 1970-01-01 00:00, Hash unknown

# backtrace
blender(BLI_system_backtrace+0x1e) [0x11c4ede]
blender() [0x8afe02]
/lib/libc.so.6(+0x33650) [0x7f042585b650]
/lib/libc.so.6(+0x9a0ed) [0x7f04258c20ed]
blender(gpu_extensions_init+0xe4) [0xe20d94]
blender(GPU_init+0x21) [0xe237f1]
blender(wm_window_add_ghostwindows+0x20a) [0x8d0e3a]
blender(WM_check+0x42) [0x8b1c92]
blender(wm_homefile_read+0x12b) [0x8bd64b]
blender(WM_init+0xe0) [0x8bf1f0]
blender(main+0xd4c) [0x89899c]
/lib/libc.so.6(__libc_start_main+0xf0) [0x7f04258485e0]
blender(_start+0x29) [0x8af6b9]
```

**Comment by rmull** [comment]
Resolved after rebuilding xorg-server, mesa3d, and nvidia, killing X, doing the gl-select dance, rebuilding glew, and restarting X. Not sure how much of that was actually necessary. I may have had a mesa3d or xorg update after which I hadn't restarted X.
